x/distribution/spec/05_hooks.md

```diff
@@ -5,26 +5,44 @@ order: 5
 # Hooks
 
 ## Create or modify delegation distribution
- 
+
  - triggered-by: `staking.MsgDelegate`, `staking.MsgBeginRedelegate`, `staking.MsgUndelegate`
 
-The pool of a new delegator bond will be 0 for the height at which the bond was
-added, or the withdrawal has taken place. This is achieved by setting
-`DelegationDistInfo.WithdrawalHeight` to the height of the triggering transaction. 
+### Before
+
+The delegation rewards are withdrawn to the delegator's withdraw address.
+The rewards include the current period, but excludes the starting period.
+Then increment the validator period, which is used for calculating rewards.
+The latter is necessary because, due to the nature of the triggers, the validator's power and share distribution can have changed.
+The reference count for the delegator's starting period is decremented.
```

Comment:
```suggestion
The rewards include the current period and exclude the starting period.
The validator period is incremented and is used for calculating rewards.
The validator period is incremented because the validator's power and share distribution might have changed.
The reference count for the delegator's starting period is decremented.
```